packages/dashboard-frontend/src/services/workspace-client/devworkspace/DevWorkspaceDefaultPluginsHandler.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { injectable } from 'inversify';
+import { api } from '@eclipse-che/common';
+import * as ServerConfigApi from '../../dashboard-backend-client/serverConfigApi';
+import { createHash } from 'crypto';
+import devfileApi from '../../devfileApi';
+import { V1alpha2DevWorkspaceSpecTemplateComponents } from '@devfile/api';
+import * as DwApi from '../../dashboard-backend-client/devWorkspaceApi';
+
+const DEFAULT_PLUGIN_ATTRIBUTE = 'default-plugin';
```

Comment:
@amisevsk wdyt? is it ok to add extra attribute in the plugin definition - https://github.com/eclipse-che/che-dashboard/pull/425/files#r768182143

Reply:
Yeah, it's not a problem. Attributes are intended to be implementation-dependant additional info, so this field is intended for use by other tools (DWO does this too). The only recommendation I'd have is to use a prefix to scope the attribute to Che itself, e.g. `che.eclipse.org/default-plugin`

Reply:
Thanks, updated to use `che.eclipse.org/default-plugin`